main.tex

```diff
@@ -769,17 +772,22 @@ \subsubsection*{Provenance entities}
 \noindent The simulation experiment is an execution of the simulation model.
 Ideally, it can be linked to a complete experiment specification (e.g., as a SED-ML~\cite{Koehn2008} or SESSL~\cite{Ewald2014} file or simply as the execution code in a general purpose programming language) and to documentation in a standard format that applies reporting guidelines such as MIASE for simulation experiments~\cite{Waltemath2011}.
 Diverse simulation experiments might be applied to serve the analysis, calibration, and validation of a simulation model.
```

Comment:
"serve" might be able to be removed? not sure what "serve" means in this context

Reply:
I've changed it to: "Diverse simulation experiments might be used for the analysis, calibration, and validation of a simulation model." Better?

Reply:
Changed "diverse", it now reads: "Different simulation experiments might be used for the analysis, calibration, and validation of a simulation model."

Reply:
Perfect!!